### PR TITLE
✅(tests) rename tests

### PR DESCRIPTION
**Pull request by SergioSim**
## Purpose

Using the same convention for all tests improves readability.

## Proposal

Renaming the tests to make them more homogeneous.

### CODE REVIEW

File: tests/backends/storage/test_fs.py [hunk 1]
```diff
@@ -54,8 +54,8 @@ def test_fs_getfile(fs):
 
 # pylint: disable=invalid-name
 # pylint: disable=unused-argument
-def test_fs_url(fs):
-    """Test that the full url of the file can be returned"""
+def test_backends_database_storage_fs_url(fs):
+    """Tests that the full url of the file can be returned."""
```
Comment (quitterie-lcs) [inline tests/backends/storage/test_fs.py:58]:
```suggestion
    """Tests that the full URL of the file can be returned."""
```

File: tests/fixtures/backends.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ class NamedClassEnum(Enum):
 
 @pytest.fixture
 def es():
-    """Create / delete an ElasticSearch test index and yield an instantiated client."""
+    """Creates / deletes an ElasticSearch test index and yield an instantiated client."""
```
Comment (quitterie-lcs) [inline tests/fixtures/backends.py:39]:
```suggestion
    """Creates / deletes an ElasticSearch test index and yields an instantiated client."""
```

File: tests/backends/storage/test_base.py [hunk 3]
```diff
@@ -3,25 +3,25 @@
 from ralph.backends.storage.base import BaseStorage
 
 
-def test_abstract_interface_with_implemented_abstract_method():
-    """Test interface mechanism with properly implemented abstract methods"""
+def test_backends_database_storage_base_abstract_interface_with_implemented_abstract_method():
```
Comment (jmaupetit) [inline tests/backends/storage/test_base.py:6]:
```suggestion
def test_backends_storage_base_abstract_interface_with_implemented_abstract_method():
```

File: tests/backends/storage/test_fs.py [hunk 4]
```diff
@@ -11,8 +11,8 @@
 
 # pylint: disable=invalid-name
 # pylint: disable=unused-argument
-def test_fs_storage_instantiation(fs):
-    """Test the FSStorage backend instantiation"""
+def test_backends_database_storage_fs_storage_instantiation(fs):
```
Comment (jmaupetit) [inline tests/backends/storage/test_fs.py:14]:
```suggestion
def test_backends_storage_fs_storage_instantiation(fs):
```

File: tests/backends/storage/test_fs.py [hunk 5]
```diff
@@ -34,8 +34,8 @@ def test_fs_storage_instantiation(fs):
 
 # pylint: disable=invalid-name
 # pylint: disable=unused-argument
-def test_fs_getfile(fs):
-    """Test that a path that exists can be returned, and throws an exception otherwise"""
+def test_backends_database_storage_fs_getfile(fs):
```
Comment (jmaupetit) [inline tests/backends/storage/test_fs.py:37]:
```suggestion
def test_backends_storage_fs_getfile(fs):
```

File: tests/backends/storage/test_fs.py [hunk 6]
```diff
@@ -54,8 +54,8 @@ def test_fs_getfile(fs):
 
 # pylint: disable=invalid-name
 # pylint: disable=unused-argument
-def test_fs_url(fs):
-    """Test that the full url of the file can be returned"""
+def test_backends_database_storage_fs_url(fs):
```
Comment (jmaupetit) [inline tests/backends/storage/test_fs.py:57]:
```suggestion
def test_backends_storage_fs_url(fs):
```

File: tests/backends/storage/test_fs.py [hunk 7]
```diff
@@ -68,8 +68,8 @@ def test_fs_url(fs):
 
 # pylint: disable=invalid-name
 # pylint: disable=unused-argument
-def test_fs_list(fs):
-    """Test archives listing in FSStorage"""
+def test_backends_database_storage_fs_list(fs):
```
Comment (jmaupetit) [inline tests/backends/storage/test_fs.py:71]:
```suggestion
def test_backends_storage_fs_list(fs):
```

File: tests/backends/storage/test_ldp.py [hunk 8]
```diff
@@ -20,8 +20,8 @@
 from ralph.exceptions import BackendParameterException
 
 
-def test_ldp_storage_instantiation():
-    """Test the LDPStorage backend instantiation"""
+def test_backends_database_storage_ldp_storage_instantiation():
```
Comment (jmaupetit) [inline tests/backends/storage/test_ldp.py:23]:
```suggestion
def test_backends_storage_ldp_storage_instantiation():
```

File: tests/backends/storage/test_ldp.py [hunk 9]
```diff
@@ -42,8 +42,8 @@ def test_ldp_storage_instantiation():
     assert isinstance(storage.client, ovh.Client)
 
 
-def test_archive_endpoint_property():
-    """Test the LDPStorage _archive_endpoint property"""
+def test_backends_database_storage_ldp_archive_endpoint_property():
```
Comment (jmaupetit) [inline tests/backends/storage/test_ldp.py:45]:
```suggestion
def test_backends_storage_ldp_archive_endpoint_property():
```

File: tests/backends/storage/test_ldp.py [hunk 10]
```diff
@@ -81,12 +81,12 @@ def test_archive_endpoint_property():
         storage._archive_endpoint
 
 
-def test_details_method(monkeypatch):
-    """Test the LDPStorage _details method"""
+def test_backends_database_storage_ldp_details_method(monkeypatch):
```
Comment (jmaupetit) [inline tests/backends/storage/test_ldp.py:84]:
```suggestion
def test_backends_storage_ldp_details_method(monkeypatch):
```

File: tests/backends/storage/test_ldp.py [hunk 11]
```diff
@@ -116,11 +116,11 @@ def mock_get(url):
     assert details.get("archiveId") == "5d49d1b3-a3eb-498c-9039-6a482166f888"
 
 
-def test_url_method(monkeypatch):
-    """Test the LDPStorage url method"""
+def test_backends_database_storage_ldp_url_method(monkeypatch):
```
Comment (jmaupetit) [inline tests/backends/storage/test_ldp.py:119]:
```suggestion
def test_backends_storage_ldp_url_method(monkeypatch):
```

File: tests/backends/storage/test_ldp.py [hunk 12]
```diff
@@ -156,11 +156,11 @@ def mock_post(url):
     )
 
 
-def test_list_method(monkeypatch):
-    """Test the LDPStorage list method with a blank history"""
+def test_backends_database_storage_ldp_list_method(monkeypatch):
```
Comment (jmaupetit) [inline tests/backends/storage/test_ldp.py:159]:
```suggestion
def test_backends_storage_ldp_list_method(monkeypatch):
```

File: tests/backends/storage/test_ldp.py [hunk 13]
```diff
@@ -193,12 +193,12 @@ def mock_list(url):
     ]
 
 
-def test_list_method_history_management(monkeypatch, fs):
-    """Test the LDPStorage list method with an history"""
+def test_backends_database_storage_ldp_list_method_history_management(monkeypatch, fs):
```
Comment (jmaupetit) [inline tests/backends/storage/test_ldp.py:196]:
```suggestion
def test_backends_storage_ldp_list_method_history_management(monkeypatch, fs):
```

File: tests/backends/storage/test_ldp.py [hunk 14]
```diff
@@ -263,8 +263,8 @@ def mock_list(url):
     )
 
 
-def test_list_method_with_details(monkeypatch):
-    """Test the LDPStorage list method with detailed output"""
+def test_backends_database_storage_ldp_list_method_with_details(monkeypatch):
```
Comment (jmaupetit) [inline tests/backends/storage/test_ldp.py:266]:
```suggestion
def test_backends_storage_ldp_list_method_with_details(monkeypatch):
```

File: tests/backends/storage/test_ldp.py [hunk 15]
```diff
@@ -319,8 +319,8 @@ def mock_get(url):
     assert list(archives) == details_responses
 
 
-def test_read_method(monkeypatch, fs):
-    """Test the LDPStorage read method with detailed output"""
+def test_backends_database_storage_ldp_read_method(monkeypatch, fs):
```
Comment (jmaupetit) [inline tests/backends/storage/test_ldp.py:322]:
```suggestion
def test_backends_storage_ldp_read_method(monkeypatch, fs):
```

File: tests/backends/storage/test_ldp.py [hunk 16]
```diff
@@ -445,8 +445,8 @@ class MockStdout:
         assert json.loads(output.read()) == archive_content
 
 
-def test_write_method_with_details():
-    """Test the LDPStorage write method"""
+def test_backends_database_storage_ldp_write_method_with_details():
```
Comment (jmaupetit) [inline tests/backends/storage/test_ldp.py:448]:
```suggestion
def test_backends_storage_ldp_write_method_with_details():
```

File: tests/backends/storage/test_swift.py [hunk 17]
```diff
@@ -11,10 +11,10 @@
 from ralph.exceptions import BackendException, BackendParameterException
 
 
-def test_swift_storage_instantiation_failure_should_raise_exception(
+def test_backends_database_storage_swift_storage_instantiation_failure_should_raise_exception(
```
Comment (jmaupetit) [inline tests/backends/storage/test_swift.py:14]: I think you can `sed` this recursively: `"s/test_backends_database_storage/test_backends_storage/g"` :sweat_smile:

File: tests/backends/storage/test_fs.py [hunk 18]
```diff
@@ -34,8 +34,8 @@ def test_fs_storage_instantiation(fs):
 
 # pylint: disable=invalid-name
 # pylint: disable=unused-argument
-def test_fs_getfile(fs):
-    """Test that a path that exists can be returned, and throws an exception otherwise"""
+def test_backends_database_storage_fs_getfile(fs):
+    """Tests that a path that exists can be returned, and throws an exception otherwise."""
```
[the remark this answers is not in the document]
Comment (SergioSim) [reply tests/backends/storage/test_fs.py:38]: Thanks)

File: tests/backends/storage/test_fs.py (hunk 1 above)
Comment (SergioSim) [reply tests/backends/storage/test_fs.py:58]: Thanks)

File: tests/fixtures/backends.py (hunk 2 above)
Comment (SergioSim) [reply tests/fixtures/backends.py:39]: Thanks)

**Review by jmaupetit — APPROVED** [review]

**Pull request merged**